api/libproject/devstart.rb

```diff
@@ -415,6 +415,49 @@ def run
   end
 end
 
+
+# Command-line parsing and main "run" implementation for deploy-api.
+class DeployApi < GcloudContext
+  def add_options
+    super
+    @parser.on(
+        "--version [VERSION]",
+        "The name of the version to deploy. Required."
+        ) do |version|
+      @opts.version = version
+    end
+    @parser.on(
+        "--promote",
+        "Use this if you want to promote this version so it receives traffic. By default, it won't."
+        ) do |promote|
+      @opts.promote = "promote"
+    end
+    @opts.promote = "no-promote" # default
+  end
+
+  def validate_options
+    super
+    if @opts.version == nil
```

Comment:
Yes, that's right. We want everything explicitly tagged. (We probably should have required it before.)

Circle should be tagging everything as circle-ci-test. This seems to work for the UI but not api right now. I'm not sure why... it's possible this will fix it.